larq/models.py

```diff
@@ -81,6 +81,27 @@ def _number_as_readable_str(num: float) -> str:
     return num + unit
 
 
+def _memory_as_readable_str(num_bits: int) -> str:
+    """Generate a human-readable string for the memory size.
+
+    1 KiB = 1024 B; we use the binary prefix (KiB) [1] instead of the decimal prefix
+    (KB) to avoid any confusion with multiplying by 1000 instead of 1024.
+
+    [1] https://en.wikipedia.org/wiki/Binary_prefix
+    """
+
+    suffixes = ["B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB"]
```

Comment:
By the time we're measuring the memory in exabytes we can be pretty sure something has gone wrong :rofl:

Reply:
Would it make sense to only include the sensible suffixes such that if something *has* gone wrong the number at least looks big (without looking at the suffix)?

Reply:
I can get behind this. Should I go up to `GiB` then? Or even just `MiB`?

Reply:
What else do you think would break @timdebruin? (Besides Zoo snapshots, which we should be able to update fairly easily when Zoo gets bumped to whatever version of Larq this is released in.)

Reply:
I am not sure, I added a test which built a network and suddenly @jamescook106 's tests broke because the model summaries had changed. But again I know very little about this which is why I tagged @jamescook106 . Maybe @AdamHillier or @lgeiger could clarify.